### DISCUSSION TITLE
does this support running multiple strategies smiultaneously

### DISCUSSION THREAD

**Discussion by isnehalkiran** [nautechsystems/nautilus_trader#766] [Q&A] [answered]
hi , i am still learning python for trading...come across this wonderful software. Very impressive work by developers. I have one query .... is running multiple strategies simultaneously or running multi-time frame strategies simultaneously supported on this platform??? 
thank you for your work

**Comment by cjdsellers** [comment, marked as answer]
Thanks for the kind feedback @isnehalkiran 

Absolutely all of that is possible, multiple venues, multiple instruments, multiple strategies, for both backtesting and live trading
